tooling/electron-mksnapshot/test/download.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { attemptDownload } from '../src/mksnapshot-download'
+import { expect } from 'chai'
+
+describe('download', () => {
+  ['12.0.10', '14.0.0-beta.3'].forEach((version) => {
+    it(`downloads providing version ${version}`, async () => {
+      const file = await attemptDownload(version, false)
+
+      expect(file.startsWith(`mksnapshot-v${version}`), `downloads correcly versioned ${file}`).to.be.true
```

Comment:
Should there be some form of mocking in this test to prevent a unit test from downloading actual electron binaries from the network? A 10s timeout on a test makes me wonder if it can be sped up somehow.

I guess a lot of useful testing of this package does require a valid download - perhaps we can mock here so at least we only download whatever single version the rest of the suite is going to use.